x/delayedack/types/rollapp_packet.go

```diff
@@ -1 +1,7 @@
 package types
+
+import rollapptypes "github.com/dymensionxyz/dymension/v3/x/rollapp/types"
+
+func GetLastestFinalizedHeigtFromStateInfo(stateInfo *rollapptypes.StateInfo) uint64 {
```

Comment:
better to have it as `(s *StateInfo)` method
could be used all over

Reply:
moved to stateinfo